console/src/view/mod.rs

```diff
@@ -164,6 +171,17 @@ impl View {
                     }
                 }
             }
+            _ => {
```

Comment:
Just realized that this is unreachable. I guess I will have to create a new `ViewState` state to indicate that the help state is present. I think as long as I store what the previous state was, I can go back to it when `esc` (or any other key) is pressed.